src/UnitTests/test_quote_prov.cpp

```diff
@@ -1,25 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "unit_test.h"
 #include "dcap_provider.h"
+#include "ext\intel\sgx_ql_lib_common.h"
 #include "local_cache.h"
-#include "UnitTests/unit_test.h"
 
+#if defined(__LINUX__)
+#include <dlfcn.h>
+#include <tgmath.h>
 #include <cstdio>
 #include <cstring>
 #include <ctime>
 #include <memory>
-#include <dlfcn.h>
-#include <tgmath.h>
-
-#include "intel/sgx_ql_lib_common.h"
+#else
+#include <iostream>
+#include <stdio.h>

```

Comment:
This is C++ code, why are you using the C runtime library functions instead of their C++ equivalents (stdio.h vs cstdio, string.h vs cstring, etc)? That allows you to have significant commonality with the Linux header block.